### PR TITLE
Alter settings

### PR DESCRIPTION
**Pull request by marikaris**

### CODE REVIEW

**Comment by marikaris** [comment]
I'll resolve the conflicts after merging  #66

File: mcmd/commands/set.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    if setting.lower() in _SETTING_SYNONYMS:
+        return _SETTING_SYNONYMS[setting.lower()]
+    else:
+        possible_settings = _get_settings()
+        settings_entity = [selected_setting for selected_setting in possible_settings if
+                           setting.lower() == selected_setting.lower()]
+        if len(settings_entity) == 1:
+            return settings_entity[0]
+        else:
+            raise McmdError('Setting: [{}] is not a valid setting'.format(setting))
```
Comment (tommydeboer) [inline mcmd/commands/set.py]:
```suggestion
            raise McmdError('Setting [{}] is not a valid settings entity'.format(setting))
```

File: mcmd/commands/set.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+# =========
+
+def arguments(subparsers):
+    p_set = subparsers.add_parser('set',
+                                  help='Alter settings',
+                                  description="Run 'mcmd set -h' to view the help for this sub-command")
+
+    p_set.add_argument('type',
+                       type=str,
+                       help="The type of setting (table extending from the sys_set_Settings table)")
```
Comment (tommydeboer) [inline mcmd/commands/set.py]:
```suggestion
                       help="Simple name of the settings entity (app, mail, opencpu, etc.) or the ID")
```

File: mcmd/commands/set.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    query = 'attrs=~id'
+    settings = _quick_get(entity, query)
+    return settings[0]['id']
+
+
+@login
+def set(args):
+    """
+    set sets the given setting of the given settings type to the given value
+    :param args: commanline arguments containing: the settings type, the setting to set, and the value to set it to
```
Comment (tommydeboer) [inline mcmd/commands/set.py]:
```suggestion
    :param args: command line arguments containing: the settings type, the setting to set, and the value to set it to
```

**Pull request merged**